redash/tasks/queries/execution.py

```diff
@@ -101,6 +101,7 @@ def enqueue_query(
                 )
 
                 logger.info("[%s] Created new job: %s", query_hash, job.id)
+                statsd_client.incr("rq.jobs.created.{}".format(queue_name))
```

Comment:
This will cover query executions and the scheduler change will cover scheduled jobs. But there are still some adhoc jobs (like sending emails) that won't be covered. Maybe we can subclass `Queue` to instrument all job creations?

Reply:
Nice, that would put all of the monitoring code into `redash/tasks/worker.py` as well. That just makes too much sense 😄